foreman-selinux-enable

```diff
@@ -22,26 +22,12 @@ fi
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
-    # Create port list cache
-    /usr/sbin/semanage port -E > $TMP_PORTS
-
-    # Remove previously defined conflicting docker_port_t (this can be removed in future release)
-    grep -E '(container|docker)_port_t' $TMP_PORTS | sed s/-a/-d/g >> $TMP_EXEC_BEFORE
```

Comment:
One comment tho before you merge, removals in SELinux can be problematic when types are assigned to ports/files. You may get a compilation error on systems where foreman_container_port_t is still assigned to a port number. At least this was what was happening in older SELinux stacks.

This patch removes the type (see below) and it could have failed on SELinux policy reload, however, this grep/sed actually removes all types which end with container_port_t which includes foreman_container_port_t therefore it should not exist anymore and reload should be successful.

Reply:
I was also a bit worried about removal, but it isn't quite clear to me what you recommend here.

Reply:
Narrator: it did, in fact, break upgrades: https://community.theforeman.org/t/luna-nightly-rpm-pipeline-354-failed/28201/3